fern/api-reference/introduction/alchemy-api-reference-overview/chain-apis-overview.mdx

```diff
@@ -981,6 +981,25 @@ Welcome to the Chain APIs hub, your centralized location for accessing all our a
 | [`eth_getFilterChanges`](/docs/node/botanix/botanix-api-endpoints/eth-get-filter-changes)                                       | [`eth_getFilterLogs`](/docs/node/botanix/botanix-api-endpoints/eth-get-filter-logs)                                         |
 | [`eth_uninstallFilter`](/docs/node/botanix/botanix-api-endpoints/eth-uninstall-filter)                                         |                                                                                             |
 
+## Hyperevm APIs
+
+📙 Get started with our [Hyperevm API Quickstart](https://www.alchemy.com/docs/reference/hyperevm-api-quickstart)
+
+| Method | Method |
```

Comment:
is it supposed to be method/method? I need to take a look at the table, not sure why there's two columns here at first glance.

Reply:
ah, found it! I think its clean, though different from the other ones.